frontend/src/events.js

```diff
@@ -46,18 +48,23 @@ const events = {
     App.update();
   },
   copy() {
+    const {exportDeckFormat: format } = App.state;
     const textField = document.createElement("textarea");
-    textField.value = filetypes.txt();
+    textField.value = exportDeck[format].copy(null, collectDeck());
```

Comment:
Is it possible to not pass a `null` here? It always seems awkward to me to pass null in methods, as it's not very clear at first glance what you're supposed to pass.

Reply:
I have the copy method take both `(filename, deck) ` in anticipation there might be a copy format which has a spot for the deck / file name.

This export format doesn't so I just didn't bother. Could be clearer to pass it in every time though

Reply:
fixed